flink-table/flink-table-planner-blink/src/test/java/org/apache/flink/table/planner/factories/TestValuesTableFactory.java

```diff
@@ -243,6 +245,19 @@ private static RowKind parseRowKind(String rowKindShortString) {
 		.asList()
 		.noDefaultValue();
 
+	private static final ConfigOption<Boolean> USE_PARTITION_PUSH_DOWN = ConfigOptions
```

Comment:
whether we can use `PARTITION_LIST` is not empty (including null) to represent `use-partition-push-down=true` ? then this config can be removed